test/test-helpers.js

```diff
@@ -16,18 +16,25 @@ define(function () {
     var self = {};
 
     // Jasmine matcher to check the log level of a log object
-    self.toBeAtLevel = function toBeAtLevel(level) {
-        var log = this.actual;
-        var expectedWorkingCalls = log.levels.SILENT - log.levels[level.toUpperCase()];
-        var realLogMethod = window.console.log;
-
-        for (var ii = 0; ii < logMethods.length; ii++) {
-            var methodName = logMethods[ii];
-            log[methodName](methodName);
-        }
-
-        expect(realLogMethod.calls.length).toEqual(expectedWorkingCalls);
-        return true;
+    self.toBeAtLevel = function toBeAtLevel() {
+        return {
+            compare: function(actual, expected){
+                var log = actual;
+                var expectedWorkingCalls = log.levels.SILENT - log.levels[expected.toUpperCase()];
+                var realLogMethod = window.console.log;
+
+                for (var ii = 0; ii < logMethods.length; ii++) {
+                    var methodName = logMethods[ii];
+                    log[methodName](methodName);
+                }
+
+                var passed = realLogMethod.calls.count() === expectedWorkingCalls;
```

Comment:
I think it's probably simpler and clearer to just put this inline as the value of 'pass' below.